git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -0,0 +1,780 @@
+</div>
+
+If you are not sure if Git is configured on your computer, enter `git config --list` into your interface. The last two lines of output should show your `user.name` and `user.email`. If these are correct you are all ready to start the module!
+
+If you need to get Git configured on your computer the [Setting Up Git](https://liascript.io/course/?https://raw.githubusercontent.com/arcus/education_modules/rmh-git-setup/git_setup/setting_up_git.md#1) module will help you get set up in about 20 minutes.
+
+## Creating a Repository
+
+Usually you will use Git to track the progress of a project. As we learn about Git, we will use the story of Wolfman and Dracula who are investigating if it
+is possible to send a planetary lander to Mars.
+
+![A collage of the project we will use as a motivating example. Figures include Dracula, a werewolf, a mummy, and several planets and moons.](media/motivatingexample.png)
+[Werewolf vs dracula](https://www.deviantart.com/b-maze/art/Werewolf-vs-Dracula-124893530)
+by [b-maze](https://www.deviantart.com/b-maze) / [Deviant Art](https://www.deviantart.com/).
+[Mars](https://en.wikipedia.org/wiki/File:OSIRIS_Mars_true_color.jpg) by European Space Agency /
+[CC-BY-SA 3.0 IGO](https://creativecommons.org/licenses/by/3.0/deed.en).
+[Pluto](https://commons.wikimedia.org/wiki/File:PIA19873-Pluto-NewHorizons-FlyingPastImage-20150714-transparent.png) /
+Courtesy NASA/JPL-Caltech.
+[Mummy](https://commons.wikimedia.org/wiki/File:Mummy_icon_-_Noun_Project_4070.svg)
+$\copyright$; Gilad Fried / [The Noun Project](https://thenounproject.com/) /
+[CC BY 3.0](https://creativecommons.org/licenses/by/3.0/deed.en).
+[Moon](https://commons.wikimedia.org/wiki/File:Lune_ico.png)
+$\copyright$; Luc Viatour / [https://lucnix.be](https://lucnix.be/) /
+[CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/deed.en).
+
+In the examples, we will see code as if it is written by Vlad Dracula. If you are practicing these steps as you move through this module, your name and email address will appear instead of Dracula's.
+
+To copy the code in the examples, enter everything after the dollar sign "$" into the command line, and then press the enter or return key. If the command has any output, it will not start with a dollar sign, but a new line where you can type after a dollar sign will appear.
+
+```console
+$logname
+vlad
+$
+```
+
+The **codeblock** above tells you that when Dracula types `logname` and presses enter, the output is 'vlad', his username. Try entering `logname` in your own console. There will be other code in front of each `$` describing who and where you are. Since the stuff before the `$` will be different for everyone, it is customary to omit it from example code. We will also omit the `$` after the output.
+
+### Initializing the repository
+
+First, let's create a directory in `Desktop` folder for our work and then move into that directory:
```

Comment:
Consider adding some instructions for folks who don't have Desktop (e.g. will this work for Windows folks?)

Reply:
Good call, I changed it to a suggestion and included a suggestion of `C` drive for Windows users.